### PR TITLE
Fix/timeline modals

### PR DESCRIPTION
**Pull request by kasperkristensen**
**What**

- Adds modals to receive returns and fulfill items for Exchanges (swaps) and Claims.
- Fixes product list in RMASelectProductsTable
- Removes the word test being displayed on items in some tables
- Adds items `Fulfilled`/`Shipped` events for claims and exchanges.
- Adds `Refund` event
- Fixes `ReceiveMenu` in the case of a return related to a `Swap` or `Claim` where return reason has already been selected,  and the case where a refund has already been issued.
- Fixes wrong padding on "Select Reason" button in RMASelectProductsTable

image

**Things to note**
- Currently receiving an swap return will result in an 500 error being thrown on the backend due to a call to a function that does not exist. https://github.com/medusajs/medusa/blob/59761163c2f0a69dd5d1c6d73c20a80c5f08e5bd/packages/medusa/src/services/swap.js#L818. An error is also thrown on line https://github.com/medusajs/medusa/blob/38762961e6d23955c297986edb01a9c5dbe21373/packages/medusa/src/api/routes/admin/orders/receive-swap.ts#L69 as the called function from orderService does not exist.
- Canceling a claim results in an error as the endpoint is defined incorrectly in medusa-react, where it is defined as /admin/orders/<claim_id>/claims//cancel
- It is currently not possible to mark claim fulfillments as shipped the fulfilment section of order details.

### CODE REVIEW

**Review by olivermrbl — CHANGES_REQUESTED** [review]
Very nice! Couple of comments here and there 👍

File: src/components/molecules/timeline-events/claim/index.tsx [hunk 1]
```diff
@@ -17,26 +18,32 @@ import ClaimDetails from "./details"
...
-const Claim: React.FC<ClaimProps> = ({ event }) => {
+const Claim: React.FC<ClaimProps> = ({ event, refetch }) => {
   const cancelClaim = useAdminCancelClaim(event.id)
   const [showDetails, setShowDetails] = useState(false)
   const [showCancel, setShowCancel] = useState(false)
+  const [showFulfillment, setShowFulfillment] = useState(false)
 
   const handleCancel = () => {
     cancelClaim.mutate("") // error on hook endpoint is /admin/orders/claim_01FTRG93AK830MS285CB20CBJW/claims//cancel
   }
```
[the remark this answers is not in the document]
Comment (kasperkristensen) [reply src/components/molecules/timeline-events/claim/index.tsx:32]: @olivermrbl Does not appear like it is working in medusa-react yet, as far as I can tell. I think we need to upgrade the version of medusa-js in medusa-react 😄

**Comment by olivermrbl** [comment]
My project seems to be breaking after having merged `develop` in. The issues seem to be related to the old Toaster. Can you start it up locally @kasperkristensen ?

**Comment by kasperkristensen** [comment]
@olivermrbl should be fixed now

**Comment by olivermrbl** [comment]
LGTM great job @kasperkristensen! But before we merge, we should get [#1041](https://github.com/medusajs/medusa/pull/1041) merged into the core first, so we can test it out 👍

**Comment by srindom** [comment]
I am not sure if this was introduced in this branch but the layered modal shows overflowing content here:

https://user-images.githubusercontent.com/7554214/154071514-2cf7648d-60e1-461c-abb5-95e315593b15.mov

**Review by srindom — APPROVED** [review]

**Pull request merged**